testcontainers/tests/images.rs

```diff
@@ -314,6 +314,29 @@ fn generic_image_with_custom_entrypoint() {
     );
 }
 
+#[test]
+fn generic_image_exposed_ports() {
+    let _ = pretty_env_logger::try_init();
+    let docker = clients::Cli::docker();
+
+    let target_port = 8080;
+
+    // This server does not EXPOSE ports in its image.
+    let generic_server = images::generic::GenericImage::new("expose_port", "latest")
```

Comment:
Maybe the image should be called `no_expose_port`? :D

Reply:
Great idea. I've made the change.